static/elements/chromedash-feature.js

```diff
@@ -111,6 +101,19 @@ class ChromedashFeature extends LitElement {
     return vendors + webdevs + standards;
   }
 
+  _getIsPreLaunch() {
+    const PRE_LAUNCH_STATUSES = [
+      'No active development',
+      'Proposed',
+      'In development',
+      'Behind a flag',
```

Comment:
Shouldn't this be 'Developer trial.'

Reply:
That change to models.py is still pending in a different CL.  However, thank you for pointing out that this will need to change after that lands.  I'll add a TODO here.